README.md

```diff
@@ -23,9 +23,42 @@ Then, instead of the `run.sh` script, run Maven directly:
 $ mvn clean install -Pnative
 ```
 
+## Using the `newdoc` script to generate new pages
+
+Che documentation follows guidelines of the Modular Documentation initiative: https://redhat-documentation.github.io/modular-docs/
+
+You can use `newdoc` to generate templates for the new documentation.
```

Comment:
I believe we should be more strongly inviting/recommending the user to use `newdoc`.